<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>242</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>pvQt -- Picture Specifications</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="filesBox" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>20</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxVisibleItems" >
    <number>7</number>
   </property>
   <property name="maxCount" >
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>File</string>
   </property>
   <property name="buddy" >
    <cstring>filesBox</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Format</string>
   </property>
   <property name="buddy" >
    <cstring>formatBox</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="formatBox" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>50</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxVisibleItems" >
    <number>6</number>
   </property>
   <property name="maxCount" >
    <number>6</number>
   </property>
  </widget>
  <widget class="QComboBox" name="faceBox" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>50</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxVisibleItems" >
    <number>7</number>
   </property>
   <property name="maxCount" >
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Face</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>90</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Display Tile Size</string>
   </property>
   <widget class="QDoubleSpinBox" name="widthDeg" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>70</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="keyboardTracking" >
     <bool>false</bool>
    </property>
    <property name="decimals" >
     <number>1</number>
    </property>
    <property name="minimum" >
     <double>10.000000000000000</double>
    </property>
    <property name="maximum" >
     <double>360.000000000000000</double>
    </property>
    <property name="value" >
     <double>90.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="heightPix" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="accelerated" >
     <bool>true</bool>
    </property>
    <property name="keyboardTracking" >
     <bool>false</bool>
    </property>
    <property name="maximum" >
     <number>20000</number>
    </property>
    <property name="singleStep" >
     <number>10</number>
    </property>
    <property name="value" >
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Degrees</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="widthPix" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="accelerated" >
     <bool>true</bool>
    </property>
    <property name="keyboardTracking" >
     <bool>false</bool>
    </property>
    <property name="maximum" >
     <number>20000</number>
    </property>
    <property name="singleStep" >
     <number>10</number>
    </property>
    <property name="value" >
     <number>500</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="heightDeg" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>70</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="accelerated" >
     <bool>true</bool>
    </property>
    <property name="keyboardTracking" >
     <bool>false</bool>
    </property>
    <property name="decimals" >
     <number>1</number>
    </property>
    <property name="minimum" >
     <double>10.000000000000000</double>
    </property>
    <property name="maximum" >
     <double>180.000000000000000</double>
    </property>
    <property name="value" >
     <double>60.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Pixels</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>115</x>
      <y>14</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>225</x>
      <y>14</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>height</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
